src/core/Elsa.Abstractions/Services/Models/StartableWorkflowDefinition.cs

```diff
@@ -1,4 +1,4 @@
 namespace Elsa.Services.Models
 {
-    public record StartableWorkflowDefinition(IWorkflowBlueprint WorkflowBlueprint, string? ActivityId, string? CorrelationId = default, string? ContextId = default);
+    public record StartableWorkflowDefinition(IWorkflowBlueprint WorkflowBlueprint, string? ActivityId, string? CorrelationId = default, string? ContextId = default, bool? isTest = default, string? signalRConnectionId = default);
```

Comment:
Core models shouldn't know about testing. Please remove both `isTest` and `signalRConnectionId`.

You can set these properties on the `WorkflowInstance` from within the `WorkflowTest.Execute` class after you received the `StartableWorkflow` object (and before you actually execute).

Keep in mind: the core library should not know anything about testing.